front/app/components/HookForm/SlugInput/index.tsx

```diff
@@ -36,18 +36,16 @@ const SlugInput = ({
           name="slug"
           type="text"
         />
-        <Text>
-          <b>
+        <Text mb={showWarningMessage ? '16px' : '0'}>
+          <i>
             <FormattedMessage {...messages.resultingPageURL} />
-          </b>
+          </i>
           : {previewUrl}
         </Text>
         {showWarningMessage && (
-          <Box mb="16px">
```

Comment:
We should avoid having margin on the "outer edges" pf a component that will imported elsewhere. The bottom margin bottom should rather be on that imported component.

I've also moved the warning message under the field, where the error would be. It's a bit weird to have a warning message pop up above a field.